src/libs/ajax.js

```diff
@@ -1338,6 +1323,35 @@ const Buckets = signal => ({
 })
 
 
+const FirecloudBucket = signal => ({
+  getServiceAlerts: async () => {
+    const res = await fetchOk(`${getConfig().firecloudBucketRoot}/alerts.json`, { signal })
+    debugger
```

Comment:
Looks like this was left in from development?

Reply:
Oops! Will fix.

Reply:
@nawatts Yes, that would be good.